module/zfs/spa.c

```diff
@@ -7063,12 +7063,13 @@ spa_vdev_attach(spa_t *spa, uint64_t guid, nvlist_t *nvroot, int replacing,
 
 	if (!replacing) {
 		/*
-		 * For attach, the only allowable parent is a mirror or the root
-		 * vdev.
+		 * For attach, the only allowable parent is a mirror or
+		 * the root vdev. A raidz vdev can be attached to, but
+		 * you cannot attach to a raidz child.
 		 */
 		if (pvd->vdev_ops != &vdev_mirror_ops &&
```

Comment:
It might be helpful to clarify in the comment that the condition is a list of permitted cases under de morgan's law.

i.e. !(A OR B OR C) === !A AND !B AND !C

If it were me, I would probably add:

> As per De Morgan's law: !(A OR B OR C) === !A AND !B AND !C

Reply:
Of course, I could just be having an off day and did not make the connection right away when reading the patch in github's viewer, which made me think expanding the comment would be helpful. If you think this is clear as is, feel free to disregard my remark.